pkg/controller/strategy/controller_test.go

```diff
@@ -30,125 +30,142 @@ func init() {
 }
 
 func TestContenderReleasePhaseIsWaitingForCommandForInitialStepState(t *testing.T) {
-	f := newFixture(t)
+	for _, i := range []uint{1, 10} {
```

Comment:
How would you feel about having a global slice with the pod counts to test that all the test cases share? That way we can put a comment there explaining why we think these specific values should be tested (mentioning the "pod math" bits).